src/s2_fos/model.py

```diff
@@ -0,0 +1,55 @@
+"""
+Implement your model here! Or not.
+"""
+import os
+
+import torch
+import numpy as np
+from transformers import (
+    AutoTokenizer, AutoConfig, AutoModelForSequenceClassification,
+)
+
+
+class PredictProbabilities:
```

Comment:
Any reason not to combine functionality with the main S2FOS class? Please do so.